web/src/index.tsx

```diff
@@ -1,24 +1,20 @@
 import React from "react";
 import { createRoot } from "react-dom/client";
-import { HashRouter } from "react-router-dom";
 import { ThemeProvider } from "styled-components";
 import { GlobalStyle } from "./styles/global-style";
 import { lightTheme } from "./styles/themes";
-import Layout from "components/layout";
+import App from "./app";
+import { BrowserRouter as Router } from "react-router-dom";
```

Comment:
We should use HashRouter just in case we want to deploy to IPFS in the future https://stackoverflow.com/questions/64546654/how-to-use-react-routing-with-ipfs

Reply:
Ok, I removed it because the react-router docs didn't recommend using HashRouter unless you have to and I didn't know ipfs would need that type of router